connectors/es/sink.py

```diff
@@ -189,10 +189,12 @@ async def run(self):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Sink...")
             raise
-        except ForceCanceledError:
-            self._logger.error(
-                f"Sink did not stop within {CANCELATION_TIMEOUT} seconds of cancelation, force-canceling the task."
-            )
+        except Exception as e:
+            if isinstance(e, ForceCanceledError) or self._canceled:
+                self._logger.warning(
+                    f"Sink did not stop within {CANCELATION_TIMEOUT} seconds of cancelation, force-canceling the task."
+                )
+                return
```

Comment:
I think you need to re-raise here, if error is not ForceCancelledError or self._canceled

Reply:
Good catch